src/lib/tabs/tab-group.scss

```diff
@@ -17,6 +17,10 @@
   position: relative;
 }
 
+.md-tab-wrap {
+  font-size: .8rem;
```

Comment:
FYI we typically want to avoid `rem` since we have no control over the root font-size.